src/pages/block/[id].tsx

```diff
@@ -246,7 +271,6 @@ export async function getServerSideProps(
     return {
       props: {
         data: {
-          block,
           blockTransactions: {
```

Comment:
Can we get block details from server side and blockTransactions we can get on client side, This is because when we will share url on any platform then we will need block details first (SEO point of view)